apps/openmw/mwrender/groundcover.cpp

```diff
@@ -199,7 +170,16 @@ namespace MWRender
          : GenericResourceManager<GroundcoverChunkId>(nullptr)
          , mSceneManager(sceneManager)
          , mDensity(density)
+         , mStateset(new osg::StateSet)
     {
+         // MGE uses default alpha settings for groundcover, so we can not rely on alpha properties
+         // Force a unified alpha handling instead of data from meshes
+         osg::ref_ptr<osg::AlphaFunc> alpha = new osg::AlphaFunc(osg::AlphaFunc::GEQUAL, 128.f / 255.f);
+         mStateset->setAttributeAndModes(alpha.get(), osg::StateAttribute::ON|osg::StateAttribute::OVERRIDE);
+         mStateset->setAttributeAndModes(new osg::BlendFunc, osg::StateAttribute::OFF|osg::StateAttribute::OVERRIDE);
+         mStateset->setRenderBinDetails(0, "RenderBin", osg::StateSet::OVERRIDE_RENDERBIN_DETAILS);
+         mStateset->setAttribute(new osg::VertexAttribDivisor(6, 1));
+         mStateset->setAttribute(new osg::VertexAttribDivisor(7, 1));
```

Comment:
And this brings back transparency?

Reply:
This basically just rearranges things so this state is set high-up in the scenegraph (and forcibly applied lower-down via `OVERRIDE`) instead of being set wherever state existed in the first place.